README.md

```diff
@@ -1,7 +1,22 @@
+## Flatpak for Pocket Casts
 
-## Updating the package
+#### Check out the upstream project here: https://github.com/FelicianoTech/pocket-casts-desktop-app
 
-Using [flatpak-builder-tools](https://github.com/flatpak/flatpak-builder-tools):
-```
-../flatpak-builder-tools/node/flatpak-node-generator.py --xdg-layout yarn /tmp/pocket-casts-desktop-app/yarn.lock
-```
+### Build the Flatpak locally
+1. Clone this repo
+2. Install the flatpak-builder package for your distro
+3. Install FreeDesktop SDK and Runtime, as well as the Electron BaseApp
```

Comment:
Maybe making the script check for missing runtimes would be better?

I've done this in https://github.com/flathub/org.ghidra_sre.Ghidra/blob/master/generate-deps.sh#L15

So you could remove step 3 and have step 4 say something like:
If necessary SDKs are missing, the script will error out and prompt you to install those.

Reply:
I was thinking about doing that so the workflow can reference it too. That should be pretty doable.

Reply:
Implemented this in local-build.sh

https://github.com/ProjectSynchro/tech.feliciano.pocket-casts/commit/e53d0a71852fdb9ee658a1d8ad7d40906b07996c